test/e2e/vsphere_node_selector.go

```diff
@@ -0,0 +1,241 @@
+/*
```

Comment:
This file should be renamed. We can add more tests for volume placement in the future.
vsphere_node_selector.go => vsphere_volume_placement.go